Azure/sonic-buildimage
# [Nokia ixs7215] Platform API 2.0 improvements

This PR provides the following platform API 2.0 improvements for the Nokia
ixs7215 platform.

- add support for Fan and PSU eeproms
- bug fixes in platform firmware status and update logic
- unit test enhancements and bug fixes
- add platform specific reboot script
- relocate pcie.yaml file



#### Why I did it
This PR has been created for the following reasons
- Improve sonic-mgmt platform test suite pass rate
- Improve coverage of platform unit tests
- Provide platform specific reboot logic as per platform porting guide
- Fix bug due to pcie.yaml file being located in the wrong directory
 
#### How I did it
Various code changes to platform API 2.0 logic

#### How to verify it
Verify all 'show platform' commands work as expected
Verify pass rate of sonic-mgmt platform test suite

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
